Loadouts/csat_katiba_ocamo.hpp

```diff
@@ -0,0 +1,226 @@
+// Author: Raeth
+// Description: CSAT: Katiba - Hex
+
+#include "undef.hpp" // Reset defines
+
+// Rifle
+#define RIFLE "arifle_Katiba_F"
+#define RIFLE_MAG "30Rnd_65x39_caseless_green:8","30Rnd_65x39_caseless_green_mag_Tracer:2"
+#define RIFLE_ATTACHMENTS "optic_ACO_grn","acc_pointer_IR"
+#define AAR_ATTACHMENTS RIFLE_ATTACHMENTS
+#define ALT_OPTICS "rhsusf_acc_compm4","rhsusf_acc_eotech_xps3","CUP_optic_CompM2_Black","CUP_optic_TrijiconRx01_black","BWA3_optic_RSAS","RKSL_optic_RMR_MS19"
+// GL Rifle
+#define GLRIFLE "arifle_Katiba_GL_F"
+#define GLRIFLE_MAG RIFLE_MAG
+#define GLRIFLE_MAG_SMOKE "1Rnd_Smoke_Grenade_shell:2","1Rnd_Smoke_Grenade_shell:2"
+#define GLRIFLE_MAG_HE "1Rnd_HE_Grenade_shell:5"
+#define GLRIFLE_MAG_FLARE "UGL_FlareYellow_F:4"
+// Carbine
+#define CARBINE "arifle_Katiba_C_F"
+#define CARBINE_MAG RIFLE_MAG
+// AR
+#define AR "LMG_Zafir_F"
+#define AR_MAG "150Rnd_762x54_Box:3"
```

Comment:
As mentioned, still not sure about having what we'd consider an MMG on the FT level.
However the 150 mags should be broken down to 100 mags
`100Rnd_762x54_Box_Tracer_Green`